script.js

```diff
@@ -0,0 +1,91 @@
+const booksWrapper = document.querySelector('.books-wrapper');
+const titleInput = document.querySelector('#new-book-title');
+const authorInput = document.querySelector('#new-book-author');
+const addBtn = document.querySelector('.add-btn');
+
+let books = [
+  {
+    title: 'Sapiens: A Brief History of Humankind',
+    author: 'Yuval Noah Harari',
+  },
+  {
+    title: 'Homo Deus',
+    author: 'Yuval Noah Harari',
+  },
+];
+
+function setStorage() {
+  localStorage.setItem('books', JSON.stringify(books));
+}
+
+function getStorage() {
+  books = JSON.parse(localStorage.getItem('books'));
+}
+
+function updateStorage() {
+  if (localStorage.length > 0) {
+    getStorage();
+  } else {
+    setStorage();
+  }
+}
+
+function addBook(title, author) {
+  const newBook = {
+    title,
+    author,
+  };
+
+  books.push(newBook);
+  setStorage();
+  renderBooks(books);// eslint-disable-line
+}
+
+function removeBook(id) {
+  books.splice(id, 1);
```

Comment:
* This is an OPTIONAL suggestion as it is not a requirement for this project but i think in your spare time you can work on this.
* Using the array method `splice` might be expensive as it mutates the original array which may lead to unexpected and hard-to-debug issues [Read more on why it is not advisable](https://blog.sapegin.me/all/avoid-mutation/)